Send activity as macro argument

Nysterian opened the pull request:
I did a quick look through the code and quickly found how to send the activity as an argument, it was pretty simple.

Nysterian committed:
Send activity as macro argument

I did a quick look through the code and quickly found how to send the activity as an argument, it was pretty simple.

Nysterian commented:
Relating to https://github.com/reshikino/activity-macro/issues/1

Lazytron commented:
I kludged a similar, uglier version at home. I found adding the argument for `item: activity.item` is technically superfluous but handy for calling the item itself without having to think about where it came from (and useful for re-used code from itemacro).

Nysterian committed:
Added item as argument

Nysterian committed:
Added concentration and results

Found out how to add a few more things, these should take care of most issues. The template should be included in the results, but I'm not sure if it handles summons.

Nysterian commented:
Ooops, had a little spacing error on this last one, but I should be done for now.
This ended up being more of a necessity for me rather than just something that "would be nice" so I looked a bit more into it.

Nysterian committed:
Update README.md

reshikino commented:
Oh, it looks good. I went to bed yesterday and thought I'd do it later. Thanks for the work!
